### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by JUN1026** [PRBonn/OverlapNet#42]
How can we get the relevant data? I only see the data structure given ：(

**Comment by NathanDrake67** [comment]
> How can we get the relevant data? I only see the data structure given ：(
yeah, I also noticed this issue. I want to know in detail how to use the Haomo dataset , though the author only  filled in his data path in config/config_haomo.yml without specifying how to use it
